### PR TITLE
Feature(CB2-8714): Auto calculate PSV laden weight

### PR DESCRIPTION
**Pull request by Daniel-Searle**
## Auto calculate PSV Brake Code (Weights)
This is to auto calculate the gross laden weight based on the manufacture year, total passengers and gross kerb weight

[CB2-8714](https://dvsa.atlassian.net/browse/CB2-8714)

## Checklist

- [ ] Branch is rebased against the latest develop/common
- [ ] Code and UI has been tested manually after the additional changes
- [ ] PR title includes the JIRA ticket number
- [ ] Squashed commits contain the JIRA ticket number
- [ ] Delete branch after merge

### CODE REVIEW

File: src/app/forms/custom-sections/weights/weights.component.ts [hunk 1]
```diff
@@ -116,4 +117,17 @@ export class WeightsComponent implements OnInit, OnDestroy, OnChanges {
...
+    const psvRecord = this.vehicleTechRecord as TechRecordType<'psv'>;
+    const techRecord_seatsUpperDeck = psvRecord?.techRecord_seatsUpperDeck ?? 0;
+    const techRecord_seatsLowerDeck = psvRecord?.techRecord_seatsLowerDeck ?? 0;
+    const techRecord_manufactureYear = psvRecord?.techRecord_manufactureYear ?? 0;
+    const techRecord_grossKerbWeight = psvRecord?.techRecord_grossKerbWeight ?? 0;
+
+    const kgAllowedPerPerson = techRecord_manufactureYear >= 1988 ? 65 : 63.5;
+
+    const totalPassengers = techRecord_seatsUpperDeck + techRecord_seatsLowerDeck + 1; // Add 1 for the driv
+    return (totalPassengers + 1) * kgAllowedPerPerson + techRecord_grossKerbWeight;
```
Comment (naathanbrown) [inline src/app/forms/custom-sections/weights/weights.component.ts:131]: have you not added one extra twice here?

**Review by naathanbrown — APPROVED** [review]

File: src/app/forms/custom-sections/psv-brakes/psv-brakes.component.html [hunk 2]
```diff
@@ -4,6 +4,7 @@
   [type]="editTypes.AUTOCOMPLETE"
   name="techRecord_brakes_brakeCodeOriginal"
   label="Brake code"
+  [hint]="'If year of manufacture is 1988, check your brake code'"
```
Comment (tomcrawleyy) [inline src/app/forms/custom-sections/psv-brakes/psv-brakes.component.html:7]: is it worth separating this string value out into an enum somewhere or would it only ever get used here?

**Review by tomcrawleyy — APPROVED** [review]

File: src/app/forms/custom-sections/psv-brakes/psv-brakes.component.html (hunk 2 above)
Comment (Daniel-Searle) [reply src/app/forms/custom-sections/psv-brakes/psv-brakes.component.html:7]: As far as I know, this is the only place this is being used

Comment (tomcrawleyy) [reply src/app/forms/custom-sections/psv-brakes/psv-brakes.component.html:7]: Cool, happy for you to hit merge then 👍

**Pull request merged**